ColorzCore/Program.cs

```diff
@@ -124,8 +127,14 @@ static void Main(string[] args)
                 Console.Error.WriteLine("No output specified for assembly.");
                 return;
             }
+            if (rawsFolder.IsNothing)
+            {
+                Console.Error.WriteLine("Couldn't find raws folder");
+                return;
+            }
             //FirstPass(Tokenizer.Tokenize(inputStream));
-            EAInterpreter myInterpreter = new EAInterpreter(game, rawsFolder, rawsExtension, inStream, inFileName, outStream, errorStream, options);
+
+            EAInterpreter myInterpreter = new EAInterpreter(game, rawsFolder.FromJust, rawsExtension, inStream, inFileName, outStream, errorStream, options);

```

Comment:
Gracefully error in the case the IO Utility cannot find the Language Raws folder (i.e. rawFolder is Nothing<string>())

Reply:
Boop you're right and I'm done.